std/root.ln

```diff
@@ -485,7 +823,7 @@ export fn set(arr: Array<bool>, idx: int64, val: bool) {
 
 // Ternary functions
 export fn pair(trueval: any, falseval: any) = new Array<any> [ trueval, falseval ];
-export fn cond(c: bool, options: Array<any>) = getR(options[1.sub(c.toInt64())]);
+export fn cond(c: bool, options: Array<any>) = getR(options[1.sub(c.toInt64()).getR()]);
```

Comment:
This `getR` won't be necessary if we adjust array accessors to accept Result-wrapped integers.